src/client.rs

```diff
@@ -26,6 +27,8 @@ pub struct Properties {
     pub sha_256: String,
     pub has_physical_mode: String,
     pub tool_version: String,
+    pub part_of: String,
```

Comment:
can you add a comment explaining this property ? I think we need to fill the comments before we don't remember what the properties are for